handle_save_climb.py

```diff
@@ -1,27 +1,21 @@
 from PySide6.QtCore import QObject, Signal
-from PySide6.QtWidgets import QDialog, QDialogButtonBox, QVBoxLayout, QLabel,\
-                              QLineEdit
 
 from open_save_clmb_dlg import open_save_clmb_dlg_box
-from open_name_and_grade_dlg_box import get_name_grade
 
 class SaveClimb(QObject):
     """Handles Saving of the climb"""
+    # climb_grade = Signal(int)
+    # climb_name = Signal(str)
+    name_grade = Signal(str, int)
     
     def handle_save_climb(self):
         
         grade, name = open_save_clmb_dlg_box()
         
         if grade and name:
-           print(f'{grade}, {name}')
+           print(f'Your route is v{grade}, and called {name}')
+        #    self.climb_grade.emit(grade)
+        #    self.climb_name.emit(name)
+           self.name_grade.emit(name, grade)
```

Comment:
i assume this saves? never used qobject before. Doesnt look like it writes to disk tho? could be wrong

Reply:
I think i need to learn signals and slots

Reply:
A signal is what the event emits and a slot is the function which handles the signal. So when I do SaveClimb.name_grade.connect(function) it will pass the signal as the arg to the slot (function). 

It's the method I'm trying to use to pass the name and grade gotten from the user and assign it to the climb object.